Runtime/Runner/Scenes/LoginController.cs

```diff
@@ -83,6 +95,8 @@ public override void Destroy()
 
         public override void Render()
         {
+            SetActive(true);
+            DontDestroyOnLoad(gameObject);
```

Comment:
Please add a comment on why "Dont Destroy On Load " is being called here.
I don't understand it.
Is the Login controller needed later?

Reply:
fixed